src/main/java/org/cyclonedx/util/ComponentWrapperSerializer.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * This file is part of CycloneDX Core (Java).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) Steve Springett. All Rights Reserved.
+ */
+package org.cyclonedx.util;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+import org.cyclonedx.model.ComponentWrapper;
+
+public class ComponentWrapperSerializer extends StdSerializer<ComponentWrapper>
+{
+  private ObjectMapper mapper;
+
+  public ComponentWrapperSerializer(final ObjectMapper mapper) {
+    this(ComponentWrapper.class);
+
+    this.mapper = mapper;
+  }
+
+  public ComponentWrapperSerializer(final Class t) {
```

Comment:
maybe generify this, like below. resolves `unchecked assignment` warning:
```patch
Index: src/main/java/org/cyclonedx/util/ComponentWrapperSerializer.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/cyclonedx/util/ComponentWrapperSerializer.java b/src/main/java/org/cyclonedx/util/ComponentWrapperSerializer.java
--- a/src/main/java/org/cyclonedx/util/ComponentWrapperSerializer.java	(revision a2fe9c4287ed6c77a602c804978c506d4b490082)
+++ b/src/main/java/org/cyclonedx/util/ComponentWrapperSerializer.java	(date 1614190644517)
@@ -36,7 +36,7 @@
     this.mapper = mapper;
   }
 
-  public ComponentWrapperSerializer(final Class t) {
+  public ComponentWrapperSerializer(final Class<ComponentWrapper> t) {
     super(t);
 
     mapper = new ObjectMapper();

```

Reply:
In: e0b54de